eradiate/kernel/tests/test_transform.py

```diff
@@ -0,0 +1,74 @@
+import enoki as ek
```

Comment:
These tests should fail (it's only by luck if they don't). Mitsuba imports should be function-scoped, and you should use a mode selection fixture for all your tests to ensure that you're always selecting the same mode. Corrections below.